lib/companion_web/jobs/process_registration.ex

```diff
@@ -44,14 +44,16 @@ defmodule Companion.Jobs.ProcessRegistration do
     msisdn <> "^^^ZAF^TEL"
   end
 
-  defp openhim_registration_from_registration(registration) do
+  defp openhim_registration_from_registration(registration, id) do
     %{
       mha: @mha_praekelt,
       swt: swt_from_channel(registration.channel),
       type: @type_nurse_registration,
       dmsisdn: registration.registered_by,
       cmsisdn: registration.msisdn,
       rmsisdn: nil,
+      eid: id,
```

Comment:
Same here with the `eid` needing to be a UUID

Reply:
noted